frontend-react/src/shared/Citation/Citation.tsx

```diff
@@ -0,0 +1,18 @@
+import { CitationProp } from "../../pages/home/HomeProps";
+
+export const Citation = ({ citation }: { citation: CitationProp }) => {
+    return (
+        <div className="padding-top-9">
+            <h3 data-testid="title" className="font-sans-lg">
```

Comment:
Are headers even semantically correct in the use-case of a citation? Are we really just wanting text of that size and should just use classes instead? It would also solve the issue of hardcoding the header levels.

Reply:
We do have headers/titles for each citation. Will update to use <p>